docker-compose.yaml

```diff
@@ -8,38 +8,87 @@ services:
     expose:
       - 5000
     tty: true
+    depends_on:
+      - db
     networks:
       app-network:
         ipv4_address: 172.22.0.6
-
   nginx:
     image: nginx:latest
-    container_name: nginx
+    container_name: nginx-service
     build:
       context: .
       dockerfile: Dockerfile.nginx
     ports:
       - "80:80"
+      - "443:443"
+    restart: always
     depends_on:
       - authz
     networks:
       app-network:
         ipv4_address: 172.22.0.5
     volumes:
       - nginx_log:/var/log/nginx
-
+      - certbot_storage:/etc/letsencrypt:rw
+  certbot:
+    container_name: certbot-service
+    build:
+      context: .
+      dockerfile: Dockerfile.certbot
+    tty: true
+    restart: always
+    expose:
+      - 80
+    volumes:
+      - certbot_storage:/etc/letsencrypt
+      - certbot_log:/var/log/letsencrypt:rw
+    depends_on:
+      - nginx
+    networks:
+      app-network:
+        ipv4_address: 172.22.0.7
+  db:
+    image: postgres:latest
+    restart: always
+    env_file:
+      - /etc/PGDATA/db_login.env
```

Comment:
Can we use default  `.env` file here. So the approach is
* create 2 files `.env.example` `.env`
* store in `.env.example` env vars that need to be set and commit it
* add .env to .gitignore and do not commit it
This way 
* when devs are testing locally they can
   * copy `.env.example` -> `.env` and adjust env vars base on their needs
   * run docker-compose locally
* we can use ansible to encrypt   .env file with credentials related to our runtime environments like dev prod, pre and deploy it to them